prow/gerrit/adapter/adapter.go

```diff
@@ -310,7 +324,8 @@ func (c *Controller) Sync() {
 					previousRun = time.Now()
 					processSingleProject(instance, project)
 				}
-			}(instance, project)
+			}(instance, project, newWorkerCount)
+			newWorkerCount++
```

Comment:
Nit: I think something like `staggerPosition` or `workerNapDuration` would be a better name.

Reply:
`staggerPosition` SGTM